pkg/templatelib/lib.go

```diff
@@ -134,5 +113,13 @@ var FuncMap = template.FuncMap{
 		} else {
 			return unsetVal
 		}
-	}),
+	})
+
+	// {{- $mungedUrl := $url | replace "git://" "https://" | trimSuffixes ".git" -}}
+	// turns: git://github.com/jsmith/some-repo.git
+	// into: https://github.com/jsmith/some-repo
+	funcMap["trimPrefixes"] = stringsActionFactory("trimPrefixes", false, stringsModifierActionFactory(strings.TrimPrefix))
+	funcMap["trimSuffixes"] = stringsActionFactory("trimSuffixes", false, stringsModifierActionFactory(strings.TrimSuffix))
```

Comment:
These support a list of prefixes/suffixes, but I'm reasonably certain we've never actually used that functionality, so all our uses _should_ be able to port to `trimPrefix` and `trimSuffix`.

Reply:
Just confirmed, `trimPrefixes` is something we've never actually used, and `trimSuffixes` is literally only used with `.git` as a single argument. :sweat_smile: